chart/dockerfiles/pgbouncer-exporter/Dockerfile

```diff
@@ -15,6 +15,7 @@
 ARG ALPINE_VERSION="3.12"
 
 FROM alpine:${ALPINE_VERSION} AS builder
+MAINTAINER "Apache Airflow Community <dev@airflow.apache.org>"
```

Comment:
This one should be removed though. Maitainer directive is deprecated and we replaced it the maintainer label which is the recommended approach: https://docs.docker.com/engine/reference/builder/#maintainer-deprecated